pkgs/development/tools/sunxi-tools/default.nix

```diff
@@ -1,11 +1,12 @@
 { lib, stdenv, fetchFromGitHub, pkg-config, libusb1, zlib }:
 
-stdenv.mkDerivation {
-  name = "sunxi-tools-20181113";
+stdenv.mkDerivation rec {
+  pname = "sunxi-tools";
+  version = "20181113";
 
   src = fetchFromGitHub {
     owner = "linux-sunxi";
-    repo = "sunxi-tools";
+    repo = ${pname};
```

Comment:
I am not a big fan of this because technically the repo name does not need to match the pname but it does often.